strings/uncle.txt

```diff
@@ -174,3 +174,35 @@ Why did that
 soda have a
 skull on it?
 -
+Something
+random just
+came up.
+-
+I'm bad at
+this. Can you
+do it for me?
+-
+Link!
+   Wake up!
+      ... Bye!
+-
+Text me when
+you hit
+go mode.
+-
+Turn off the
+stove before
+you leave.
+-
+It's raining.
+Umbrella on
+Pedestal.
```

Comment:
This will probably confuse people into thinking pedestal is required, let's cut this one.

Reply:
I removed the location.  Uncle is taking the umbrella.  Link will have to adventure in the rain.